### PR TITLE
Added link to botkit docs

### PR DESCRIPTION
**Pull request by jasonrhodes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
 
 ## Composable Botkit Bots
 
-Skellington is a skeleton for your [Botkit](https://github.com/howdyai/botkit) bots. It handles the boilerplate connection
```
Comment (jasonrhodes) [inline README.md]: Woops, atom.io being aggressive with the whitespace edits there, muddying up the PR like a n00b

**Comment by colestrode** [comment]
Whoooops, looks like i updated the readme under you... mind merging in master?

**Comment by jasonrhodes** [comment]
Look ma, no merge!

**Comment by colestrode** [comment]
git blows my mind sometimes

**Pull request merged**